openapi_spec_validator/handlers.py

```diff
@@ -3,17 +3,20 @@
 
 from six.moves.urllib.parse import urlparse
 from six.moves.urllib.request import urlopen
-from yaml import safe_load
+from yaml import load
+
+from openapi_spec_validator.loaders import ExtendedSafeLoader
 
 
 class UrlHandler:
     """OpenAPI spec validator URL scheme handler."""
 
-    def __init__(self, *allowed_schemes):
+    def __init__(self, *allowed_schemes, loader=ExtendedSafeLoader):
```

Comment:
looks like this syntax breaks python2.7. maybe allow overriding the loader with `**kwargs` instead?

Reply:
Yup, hat's the only way to support py2